sdk/mongodbatlas/Azure.ResourceManager.MongoDBAtlas/api/Azure.ResourceManager.MongoDBAtlas.net8.0.cs

```diff
@@ -0,0 +1,213 @@
+        public virtual Azure.Response<bool> Exists(string organizationName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<bool>> ExistsAsync(string organizationName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.MongoDBAtlas.OrganizationResource> Get(string organizationName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Pageable<Azure.ResourceManager.MongoDBAtlas.OrganizationResource> GetAll(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.AsyncPageable<Azure.ResourceManager.MongoDBAtlas.OrganizationResource> GetAllAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.MongoDBAtlas.OrganizationResource>> GetAsync(string organizationName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.NullableResponse<Azure.ResourceManager.MongoDBAtlas.OrganizationResource> GetIfExists(string organizationName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.NullableResponse<Azure.ResourceManager.MongoDBAtlas.OrganizationResource>> GetIfExistsAsync(string organizationName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        System.Collections.Generic.IAsyncEnumerator<Azure.ResourceManager.MongoDBAtlas.OrganizationResource> System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.MongoDBAtlas.OrganizationResource>.GetAsyncEnumerator(System.Threading.CancellationToken cancellationToken) { throw null; }
+        System.Collections.Generic.IEnumerator<Azure.ResourceManager.MongoDBAtlas.OrganizationResource> System.Collections.Generic.IEnumerable<Azure.ResourceManager.MongoDBAtlas.OrganizationResource>.GetEnumerator() { throw null; }
+        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+    }
+    public partial class OrganizationResourceData : Azure.ResourceManager.Models.TrackedResourceData, System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.MongoDBAtlas.OrganizationResourceData>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.MongoDBAtlas.OrganizationResourceData>
```

Comment:
this name is quite common, could we rename this to something like `MongoDBAtlasOrganization`?
To do this, you need to open a PR to make changes in your `client.tsp` to add some renaming clause such as:
```typespec
@@clientName(OrganizationResource, "MongoDBAtlasOrganization", "csharp");
```
and you could generate the library by using a local spec repo:
```
dotnet build /t:GenerateCode /p:LocalSpecRepo="absolutePathOfSpecRepo"
```
and during the review process, we will review both this PR and the corresponding spec PR.
When everything is ready, we will merge your spec PR for the client.tsp first, and then you could update the `tsp-location.yaml` file to the new commit id.